src/handlers/major_change.rs

```diff
@@ -587,11 +585,11 @@ async fn process_seconded(
             .post_comment(
                 &ctx.github,
                 &*format!(
-                    "The final comment period is now complete, this major change is now **accepted**.\
-                    \
-                    As the automated representative, I would like to thank the author for their work and everyone else who contributed to this major change proposal.\
-                    \
-                    *If you think this major change shouldn't have been accepted, feel free to remove the `{}` label and reopen this issue.*",
+r#"The final comment period is now complete, this major change is now **accepted**.
+
+As the automated representative, I would like to thank the author for their work and everyone else who contributed to this major change proposal.\n\
```

Comment:
No need for `\n\` since it's now a `r` string.

Reply:
Weird, must a wrong rebase. Fixed.